File: source/examples/Relationships/ToMany/ToMany.cs
public class Person : RealmObject
{
    public String Name { get; set; }
    public Date Birthdate { get; set; }
    public IList<Dog> Dogs { get; set; }
}

public class Dog : RealmObject
{
    public string Name { get; set; }
    public int Age { get; set; }
    public string? Breed { get; set; }
}

File: source/examples/Relationships/ToMany/ToMany.java
public class Person extends RealmObject {
    @Required
    public String name;

    @Required
    public Date birthdate;

    public RealmList<Dog> dogs;
}

public class Dog extends RealmObject {
    @Required
    public String name;

    @Required
    public int age;

    public String breed;
}
